src/LinuxTracing/PerfEvent.h

```diff
@@ -154,6 +154,15 @@ class LostPerfEvent : public PerfEvent {
   uint64_t GetStreamId() const { return ring_buffer_record.sample_id.stream_id; }
 
   uint32_t GetCpu() const { return ring_buffer_record.sample_id.cpu; }
+
+  void SetPreviousTimestamp(uint64_t previous_timestamp) {
+    previous_timestamp_ = previous_timestamp;
+  }
+  uint64_t GetPreviousTimestamp() const { return previous_timestamp_; }
```

Comment:
Nit: nodiscard

Reply:
Done (but note how 95% of this file misses it, it's a very old file).